src/hubmap_translator.py

```diff
@@ -21,7 +21,7 @@
 from translator.tranlation_helper_functions import *
 from translator.translator_interface import TranslatorInterface
 
-logging.basicConfig(format='[%(asctime)s] %(levelname)s in %(module)s: %(message)s', level=logging.DEBUG,
+logging.basicConfig(format='[%(asctime)s] %(levelname)s in %(module)s: %(message)s', level=logging.INFO,
```

Comment:
We also use the DEBUG level for troubleshooting on the indices config for deployment. Please revert.

Reply:
Logging level reverted to DEBUG in branch file.